browse-scaler/src/cache.js

```diff
@@ -34,19 +53,19 @@ exports.cacheImage = (key, image) => {
  * @returns {Buffer<Image>} the image associated with given cache key or null if none is found
  */
 exports.getImageFromCache = async key => {
-  try {
-    const image = await getAsync(key);
-
-    // workaround for bad cache entries that have the string "null" for the value
-    if (image && image !== 'null') {
-      console.log('got image from cache');
-      return image;
-    }
-
-    console.log('image is not in cache');
-    return null;
-  } catch (err) {
-    console.error(`Could not get image from cache: ${err}`);
-    return null;
-  }
+  let client = getClient();
+  return promisify(client.get).bind(client)(key)
+    .catch(err => {
+      console.error("Unable to retrieve image from Redis.");
+      return null;
+    })
+    .then(image => {
+      // workaround for bad cache entries that have the string "null" for the value
+      if (image && image !== 'null') {
+        console.log('got image from cache');
+        return image;
+      }
+      console.log('image is not in cache');
```

Comment:
This might help with troubleshooting
```suggestion
      console.log(`image is not in cache for concept: ${key}`);
```

Reply:
done